docs/user-guide/configure-apiml-zosmf-workflow.md

```diff
@@ -0,0 +1,113 @@
+# Configuring API ML with z/OSMF Workflows
+
+After you install Zowe, you can register and execute the z/OSMF workflows in the web interface to perform a range of
+API ML/Zowe configuration tasks. z/OSMF helps to simplify the API ML configuration tasks and does not require the level of
+expertise that is needed to perform manual API ML/Zowe configuration. This configuration method also runs the `zwe install` and `zwe init`
+command to initialize Zowe z/OS runtime.
+
+:::info Required role: system programmer
+:::
+
+Ensure that you meet the following requirements before you start your Zowe configuration:
+
+- Install and configure z/OSMF
+- Install Zowe with an SMP/E build, PSWI, or a convenience build
+
+You can complete the following tasks with the **Configuration of API ML/Zowe 3.0** workflow:
+
+- Configure the Zowe instance with simplified and optimized setup of API ML
+
+Following components are automatically enabled:
+- API Gateway
+- X509
+- ZAAS
+- API Catalog
+- Discovery service
+- Caching service
+
+Following components are automatically disabled, could be enabled only via manual change in yaml file (not recommended for simplified and optimized configuration of API ML):
+- Application server
+- ZSS component
+- JES Explorer
+- MVS Explorer
+- USS Explorer
+
+You can execute the Zowe configuration workflow either from a PSWI during deployment or later from a created software
+instance in z/OSMF. Alternatively, you can execute the configuration workflow z/OSMF during the workflow registration
+process.
+
+The configuration workflow described in this article is executed directly from a deployment of the Zowe PSWI.
+
+## Execute Configuration of Zowe 3.0 workflow from PSWI
```

Comment:
Is this the correct name of the workflow or should this be API ML specific?

Reply:
API ML specific would be probably better in here, so instead of "Configuration of Zowe 3.0"  to: Stand-alone Zowe API ML Configuration

Reply:
updated